packages/core/css/type.module.css

```diff
@@ -1,6 +1,7 @@
 :root {
-  --psTypeFontFamily: "Gotham SSm A", "Gotham SSm B", sans-serif;
-  --psTypeFontFamilyCode: "Source Code Pro", monospace;
+  /* prettier-ignore */
+  --psTypeFontFamily: 'PS TT Commons', 'Gotham SSm A', 'Gotham SSm B', sans-serif;
```

Comment:
👍 nice. didn't think about falling back to gotham but good idea.

@sweeting i'm wondering what the ie11 fallback should be

Reply:
Thought we had said that IE gets sans-serif.

Reply:
we did. i was just wondering if there was something more specific. like 'Arial'...

Reply:
Gotcha. I like going straight to sans-serif just to get the system font that's OS-specific.  

That's what `sans-serif` is to me conceptually, but I did also see that there's a new `system-ui` option, which I don't yet understand how it differs.

Reply:
Arial might be a less noticeable difference as fallback for IE 11 than the Windows System font Segoe UI (a humanist sans).

![Sample Page - PS TT Commons](https://user-images.githubusercontent.com/48542/100909368-dd20a480-3489-11eb-8ec5-a43e15d81efb.png)

![Sample Page - Segoe UI](https://user-images.githubusercontent.com/48542/100909380-e1e55880-3489-11eb-8f2f-58d4f6402add.png)

![Sample Page - Arial](https://user-images.githubusercontent.com/48542/100909389-e3af1c00-3489-11eb-9367-b5af323860dd.png)